app/models/minion.rb

```diff
@@ -106,6 +106,11 @@ def self.mark_pending_bootstrap
                                            Minion.highstates[:failed]])
           .update_all highstate: Minion.highstates[:pending]
   end
+
+  # Forcefully updates all nodes to a pending highstate
+  def self.mark_pending_bootstrap!
+    Minion.update_all highstate: Minion.highstates[:pending]
```

Comment:
There are some minions that might not have roles assinged (pending acceptance), we shouldn't update those, so this should be: `Minion.assigned_role.update_all highstate: Minion.highstates[:pending]`

Reply:
updated, thanks